### PR TITLE
Adding 'Capitalize Parsons' to best practices issue #286

### PR DESCRIPTION
**Pull request by ehrenfoss**
Small change, apologies for all the unnecessary commits in my fork

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -110,6 +110,8 @@ The following is a list of best practices to consider when writing code for the
 
 * Inline comments explaining complex codes and methods are appreciated.
 
+* Capitalize Parsons
```
Comment (eliotst) [inline CONTRIBUTING.md:113]: @ehrenfoss Thanks for working through this documentation stuff. Could you qualify this to say something about how contributors should capitalize Parsons in documentation and comments?

**Comment by shaunagm** [comment]
@ehrenfoss want to pair on finishing this?  I can help you with the git commands needed to squash down all your commits into a single commit.  I know git can be a pain to learn so I'm happy to work with you rather than giving you a list of commands to run (but I can give you the list of commands if you prefer).

**Comment by ehrenfoss** [comment]
@shaunagm sorry I let this linger. Yes, that would be helpful. I started playing around with `git rebase -i` and it feels like a great way to from messy to even more messy without some guidance.

**Comment by ehrenfoss** [comment]
@shaunagm I committed & pushed just the capitalization change above. I don't see in this PR that the remaining commits were squashed/discarded however. Thanks for your help!

**Comment by shaunagm** [comment]
Can you try a [force push](https://stackoverflow.com/a/12610763/1026275)?  You made need to force-push to override the previous set of commits.  I'm not entirely sure how that works with Github's auto-detection of pull-requests/updates to pull requests, but it works for me when I rebase and need to overwrite a PR coming from a branch.

**Comment by ehrenfoss** [comment]
Tried that, it said everything is up to date. My fork doesn't show the squashed commits either. Maybe something wasn't pushed from the reset to a specific commit that we did?

**Comment by shaunagm** [comment]
Can you try to push again, and take a screenshot of the result?

**Comment by ehrenfoss** [comment]
```
git rebase -i origin/master   <-- resulted in a 'noop' interactive editor
git fetch upstream  <-- lots of updates, ran clean with no conflicts
git merge upstream/master
git push -u origin master
```

Are we GTG? I see the proper change in the Files Changed tab. I'm not sure what else I should do to further merge commits. @shaunagm

**Comment by shaunagm** [comment]
I'd really prefer not to put these extra 13 commits into master.  Not sure why the commits aren't disappearing with force push to your fork's master.  What command did you run?  We might have to just close this PR and do a completely new one.

**Comment by ehrenfoss** [comment]
I'll nuke this PR. Force push just says `Everything up-to-date`

**Pull request closed without merging**